tests/acceptance/percy/snapshots.mjs

```diff
@@ -0,0 +1,32 @@
+import puppeteer from 'puppeteer';
+import percySnapshot from '@percy/puppeteer';
```

Comment:
Is there a difference between using @percy/script (PercyScript setup from theme) vs @percy/puppeteer?

Reply:
they've very recently deprecated percyscript https://docs.percy.io/docs/percyscript
percyscript is basically puppetter with some VERY thing wrapping, so it should be incredibly easy to migrate
I mean just looking at the code there's almost no differences